src/iOS.Autofill/Utilities/BaseLoginListTableSource.cs

```diff
@@ -80,6 +307,31 @@ private async Task SelectRowForPasskeyCreationAsync(UITableView tableView, NSInd
 
             Context.PickCredentialForFido2CreationTcs.SetResult((item.Id, null));
         }
+
+        private async Task<CipherViewModel> DeselectRowAndGetItemAsync(UITableView tableView, NSIndexPath indexPath)
+        {
+            tableView.DeselectRow(indexPath, true);
+            tableView.EndEditing(true);
+
+            var item = Items.ElementAtOrDefault(GetIndexForItemAt(tableView, indexPath));
```

Comment:
❓ Items null check?

Reply:
If `null` it's ok to break here so we know something is wrong.